java/kotlin-extractor/src/main/kotlin/KotlinUsesExtractor.kt

```diff
@@ -1506,13 +1520,18 @@ open class KotlinUsesExtractor(
      */
     fun getValueParameterLabel(vp: IrValueParameter, parent: Label<out DbCallable>?): String {
         val declarationParent = vp.parent
-        val parentId = parent ?: useDeclarationParent(declarationParent, false)
+        val overriddenParentAttributes = (declarationParent as? IrDeclaration)?.let {
+            overriddenFunctionAttributes[it]
+        }
+        val parentId = parent ?: overriddenParentAttributes?.id ?: useDeclarationParent(declarationParent, false)
```

Comment:
Would this work with recursive calls too? `overriddenFunctionAttributes` only has a reference to one overload at a time. Could we need multiple at any time? For example when generating the overload `f(String,String)` would we need access to both `f(String,String)` and `f(String)`?
```
object X {
    @JvmStatic
    @JvmOverloads
    fun f(s0: String = f("", ""), s1: String = f(""), s2: String = "") = ""
}
```

Reply:
Answering this to myself: I think this question doesn't make sense. We're accessing parameters here, I don't think it's possible to access parameters of multiple overloads, we can at most access parameters that are declared earlier in the parameter list, and those will come from the current declaration.